godot/ui/theme.tres

```diff
@@ -1471,7 +1471,7 @@ corner_radius_bottom_right = 3
 corner_radius_bottom_left = 3
 corner_detail = 5
 
-[sub_resource type="Image" id="Image_6gtp2"]
+[sub_resource type="Image" id="Image_c0sur"]
```

Comment:
This is one of those examples I was talking about where the editor changed these values and I have no idea why. :smile: